Documentation/Getting-Started/quickstart.md

```diff
@@ -57,6 +57,9 @@ The first step is to deploy the Rook operator.
 !!! note
     These steps are for a standard production Rook deployment in Kubernetes. For Openshift, testing, or more options, see the [example configurations documentation](example-configurations.md).
 
+!!! note
+    Rook images are available in multiple container repositories. For dockerHub it is available under https://hub.docker.com/repository/docker/rook/ceph/tags org, for Quay it is available under https://quay.io/repository/rook/ceph and for ghcr images are available under https://github.com/orgs/rook/packages/container/package/ceph.
```

Comment:
```suggestion
    Rook images are available in multiple container repositories. The examples by default reference [DockerHub](https://hub.docker.com/repository/docker/rook/ceph/tags). Images are also pushed to [Quay](https://quay.io/repository/rook/ceph?tab=tags) and [GHCR](https://github.com/orgs/rook/packages/container/package/ceph).
```

Reply:
The link for [GHCR](https://github.com/orgs/rook/packages/container/package/ceph) is not working, I guess it will be working after the first push from master?

Reply:
Yess